.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  pull_request: # all pull requests
+  push:
+    branches:
+      - master
+
+jobs:
+  hotspot:
+    strategy:
+      matrix:
+        java: [ '8' ] # start slow, then add: , '9', '10', '11', '14'
+    runs-on: 'ubuntu-latest'
+    env:
+      MVN_CMD: ./mvnw --no-transfer-progress -B
+    name: java-${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - name: Set up JDK
+        uses: actions/setup-java@v2.4.0
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Build
+        run: ${{env.MVN_CMD}} verify
+      - name: Code coverage
+        if:  ${{matrix.java}} == '8'
+        run: ${{env.MVN_CMD}} clean clover:setup test && ${{env.MVN_CMD}} -pl . clover:clover clover:check coveralls:report;
+
+  java-7:
```

Comment:
What about the `export MAVEN_OPTS="-Dhttps.protocols=TLSv1.2 -Xmx512m -XX:MaxPermSize=128m"` line in the current travis build?  It'll fail without the TLSv1.2 flag, no?

Reply:
yeah, I missed that line 🤦